server/routes/induction/update/qualificationsListUpdateController.ts

```diff
@@ -15,6 +15,18 @@ export default class QualificationsListUpdateController extends QualificationsLi
 
   getBackLinkUrl(req: Request): string {
     const { prisonNumber } = req.params
+    const { pageFlowHistory } = req.session
+    if (pageFlowHistory) {
+      // TODO - this needs thinking through!
```

Comment:
I got to this late last night and couldn't test it due to the server instances being shutdown! 😂 However, that may be a blessing in disguise as this feels somewhat messy/brittle. I'll have a fresh look at this tomorrow 👍